packages/template-retail-react-app/app/pages/cart/index.jsx

```diff
@@ -86,6 +86,77 @@ const Cart = () => {
 
     const {data: customer} = useCurrentCustomer()
     const {customerId, isRegistered} = customer
+
+    /***************** Product Bundles ************************/
+
+    // We use the `products` object to reference products by itemId instead of productId
+    // Since with product bundles, even though the parent productId is the same,
+    // variant selection of the bundle children can be different,
+    // and require unique references to each product bundle
+    const [productsByItemId, setProductsByItemId] = useState({})
```

Comment:
Is this state used only for product bundles? Or for any other kinds of products?

Reply:
This is currently used for all products within the basket